.travis.yml

```diff
@@ -17,22 +19,25 @@ before_script: &before_script
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - rvm: 2.6
   include:
-    - rvm: 2.4
+    - rvm: 2.5
       env:
         - SPECS=spec/controllers/api/v1/[a-m]*.rb
-    - rvm: 2.4
+    - rvm: 2.5
       env:
         - SPECS=spec/controllers/api/v1/[n-s]*.rb
-    - rvm: 2.4
+    - rvm: 2.5
       env:
         - SPECS=spec/controllers/api/v1/[t-z]*.rb
-    - rvm: 2.4
+    - rvm: 2.5
       env:
         - SPECS="spec/controllers/**.rb spec/controllers/api/*.rb spec/models spec/operations spec/counters spec/routes"
-    - rvm: 2.4
+    - rvm: 2.5
       env:
         - SPECS="spec/lib spec/workers spec/serializers spec/services spec/requests spec/middleware spec/mailers spec/policies"
+    - rvm: 2.6
```

Comment:
ruby 2.6 and rails 4 don't play nicely together, https://github.com/rails/rails/issues/34790#issuecomment-450190024

That is won't fix so these tests will always fail until we bump to rails 5. Probably not worth testing this now until we've done the upgrade.